macros/clone_schema.sql

```diff
@@ -0,0 +1,266 @@
+{%- macro clone_schema(schema_one, schema_two, comment_tag='') -%}
+
+    {#/* Copies tables, sequences and views from `schema_one` to `schema_two` if `comment_tag` isn't specified. If `comment_tag` argument is specified, it copies only tables and sequences that have `comment` metadata field equal to the passed value of `comment_tag` argument.
+       ARGS:
+         - schema_one (string) : name of first schema.
+         - schema_two (string) : name of second schema.
+         - comment_tag (string) : value of `comment` metadata field that indicates either table or sequence for copying. If it's not specified, all objects from `schema_one` will be copied to `schema_two`.
+       RETURNS: nothing to the call point.
+       SUPPORTS:
+            - Postgres
+            - Snowflake
+    */#}
+
+    {%- if target.type == 'postgres' -%}
+
+        {% set set_pg_function %}
+
+            CREATE OR REPLACE FUNCTION clone_schema(source_schema text, 
```

Comment:
I've found that there are neither special clone schema command or procedure in Postgres. Finally I've decided to make this job via single crate/alter commands that are formed while iteration on lists of original objects in loops. This part of macro is based on [this script](https://www.postgresql.org/message-id/CANu8FiyJtt-0q%3DbkUxyra66tHi6FFzgU8TqVR2aahseCBDDntA%40mail.gmail.com) from Internet.

Reply:
Code style of this part are questionable and might not clear - I've left a lot of original artefacts and mechanisms here. I can rewrite it more clearly but suppose such sort of work a bit redundant for now. At least, according to tests it works%)